src/ex1/script.js

```diff
@@ -0,0 +1,82 @@
+let pendingTaskCount = 0;
+
+const taskInput = document.getElementById("todoTaskInput");
+const clearButton = document.getElementById("clearButtonId");
+
+taskInput.addEventListener("keypress", function(event) {
+    if (event.key === "Enter") {
+        event.preventDefault();
+        document.getElementById("addTaskButtonId").click();
+    }});
+
+if(pendingTaskCount === 0){
+    startingPInsert();
+    displyPendingTasks();
+}
+
+function startingPInsert(){
```

Comment:
what's a "P"? would you be able to guess it in a week from today? :) think about yourself needs to manage this code later on. names are super important for clarity